packages/orama/src/components/sorter.ts

```diff
@@ -103,56 +101,101 @@ async function insert(
   if (!sorter.enabled) {
     return
   }
+
+  sorter.language = language
```

Comment:
I don't think we want to support a language change during the insertion: the language remains the same as the DB creation.

Reply:
This is what Orama does already: https://github.com/oramasearch/orama/blob/main/packages/orama/src/components/sorter.ts#L111

And we don't have access to Orama instance, so this could land a breaking change, so I don't think that is worth the issue of doing that.